opds_import.py

```diff
@@ -607,10 +636,12 @@ def summary_to_linkdata(detail):
             data_source_last_updated=last_opds_update,
         )
             
-        # Only add circulation data if the data source is lendable.
-        if data_source.offers_licenses:
+        # Only add circulation data if both the book's distributor *and*
+        # the source of the OPDS feed are lendable data sources.
+        if (circulation_data_source and circulation_data_source.offers_licenses
+            and metadata_data_source.offers_licenses):
```

Comment:
Should we change the content server data source back to not offering licenses? And then we wouldn't need to check the metadata_data_source here.